charts/plgd-hub/templates/resource-aggregate/config.yaml

```diff
@@ -45,16 +45,24 @@ data:
         nats:
           url:{{ printf " " }}{{- include "plgd-hub.natsUri" (list $ .clients.eventBus.nats.url) | quote }}
           flusherTimeout: {{ .clients.eventBus.nats.flusherTimeout | quote }}
-          jetstream: {{ .clients.eventBus.nats.jetstream }}
           tls:
             {{- $natsTls := .clients.eventBus.nats.tls }}
             {{- include "plgd-hub.internalCertificateConfig" (list $ $natsTls $resourceAggregateCertPath) | indent 10 }}
             useSystemCAPool: {{ .clients.eventBus.nats.tls.useSystemCAPool }}
+          jetstream: {{ .clients.eventBus.nats.jetstream }}
+          {{- $leadResourceType := .clients.eventBus.nats.leadResourceType | default $.Values.global.nats.leadResourceType }}
+          {{- if $leadResourceType }}
+          leadResourceType:
+            enabled: {{ $leadResourceType.enabled | default false }}
+            regexFilter: {{ $leadResourceType.regexFilter }}
```

Comment:
if regexFilter is array this could be a issue - you need to use {{ range $leadResourceType.regexFilter }}

Reply:
fixed